src/nvim/tui/input.c

```diff
@@ -390,9 +405,15 @@ static bool handle_bracketed_paste(TermInput *input)
       // Paste phase: "disabled".
       input->paste = 0;
     }
-    return true;
+    return kComplete;
+  } else if (buf_size < 6
```

Comment:
`but_size < 6` looks like "extra" condition?
 or this is done intentionally, for readability?

Reply:
Without this condition, receiving an input like `"\x1b[20000"` will cause unnecessary delay. It is unlikely that you will receive such input, but it is possible that you will receive such input if you paste without bracketed paste.

Reply:
@erw7 it makes sense